src/data/StringListOperations.cpp

```diff
@@ -26,15 +32,15 @@ StringListOperations::StringListOperations()
 }
 
 void
-StringListOperations::Parse(const char* start, const char* end)
+StringListOperations::Parse(std::string_view str)
 {
 	uint32_t pendingOperation = 0;
-	StringPart* pendingParameter = nullptr;
+	std::string_view* pendingParameter;
```

Comment:
I think you cannot remove the assignment to nullptr. Or else the pointer is filled with random junk from the stack. Or is this not the case anymore in C++20?

Reply:
Ah yes, that was a mistake.